### PR TITLE
Aws deploy

### PR DESCRIPTION
**Pull request by cpfergus1**
## Description

Deploy to AWS

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] I have updated the README to reflect any changes.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -50,3 +50,39 @@ jobs: # a collection of steps
       - store_test_results: # Upload test results for display in Test Summary: https://circleci.com/docs/2.0/collect-test-data/
           path: test_results
       # See https://circleci.com/docs/2.0/deployment-integrations/ for example deploy configs
+  deploy:
+      docker:
+        - image: circleci/python:2.7
```
Comment (ShaunDaneJames) [inline .circleci/config.yml:55]: Oh neat, we are using python for the docker image!

File: .circleci/config.yml [hunk 2]
```diff
@@ -50,3 +50,39 @@ jobs: # a collection of steps
...
+              sudo pip install awsebcli --upgrade
+        - run:
+            name: Create AWS credentials manually
+            command: |
+              mkdir ~/.aws
+              touch ~/.aws/config
+              chmod 600 ~/.aws/config
+              echo "[profile eb-cli]" > ~/.aws/config
+              echo "aws_access_key_id=$AWS_ACCESS_KEY_ID" >> ~/.aws/config
+              echo "aws_secret_access_key=$AWS_SECRET_ACCESS_KEY" >> ~/.aws/config
```
Comment (ShaunDaneJames) [inline .circleci/config.yml:71]: Are these environment variables the team will need ($AWS_ACCESS_KEY_ID, $AWS_SECRET_ACCESS_KEY)?

**Review by ShaunDaneJames — APPROVED** [review]
Looks great! Did we have a production URL?

File: .circleci/config.yml (hunk 1 above)
Comment (BJSherman80) [reply .circleci/config.yml:55]: Did you have to download pythin to your local machine?

**Review by BJSherman80 — APPROVED** [review]

**Pull request merged**